src/components/Card/Card.tsx

```diff
@@ -100,46 +112,69 @@ export class Card extends React.Component<CardProps, CardState> {
       id: this.props.id,
       vote: -1
     });
+  }
 
-    this.setState({votes: this.state.votes - 1});
+  renderUserSentiment() {
+    return (
+      <span className="sentiment">
+        Your Vote: {this.state.userSentiment > 0 ? `+${this.state.userSentiment}` : this.state.userSentiment}
+      </span>
+    );
+  }
+
+  renderResults() {
+    return (
+      <>
+        <span className="vote-count">Votes:{this.state.votesCount}</span>
+        <span className="sentiment">
+          Sentiment: {this.state.netSentiment > 0 ? `+${this.state.netSentiment}` : this.state.netSentiment}
```

Comment:
Am I reading this correctly in that "sentiment" can go below 0? Essentially, cards/retro items can be downvoted into negative sentiment?

Reply:
This has now been changed with the introduction of "stars" which cannot go below 0. You can also not remove other people's stars.